### ISSUE TITLE
Errors loading tracks 3 & 4

### ISSUE THREAD

**Issue by sysads** [KozzX/MinigameRacerCorona#1]
Getting following errors while loading tracks 3 & 4:

\minigameracercorona-master\minigameracer\corona-master\scripts\cenas\3tracks\normal.lua:96: in function <c:\users\sysads\downloads\minigameracercorona-master\minigameracercorona-master\scripts\ce
nas\3tracks\normal.lua:30>
        ?: in function 'dispatchEvent'
        ?: in function <?:865>
        (tail call): ?
        ?: in function <?:498>
        ?: in function <?:221>

**Comment by KozzX** [comment]
Thanks for the issue report. The only game scene that is rellay working so far and is still under modifications is the 2 Tracks/ Easy. When my result scene is done and the game scene is ready I will start to work on the other scenes.
Thanks

**Comment by sysads** [comment]
Thanks for the update and great game. Will I be permitted to use it when its done? Also I am getting this message "premium graphics feature (object.stroke)". Do I need to have an enterprise license to see what it does in the game?

**Comment by KozzX** [comment]
You mean using my code when its done? About the premium message, I just use the pro version of corona that is free now.

**Comment by sysads** [comment]
Yes if I can use your code. Is the pro version of corona only for mac or is there a version for Windows?

**Comment by KozzX** [comment]
I dont see any problem of using the base code. When I'm done I want to create a series explaining the entire development of this game. The pro version of corona is for both windows and mac. If you are already logged in on corona sdk, you have to deauthorize your user on your pc and log in again, it will automatically upgrade to pro version. If you have some question, feel free to ask.
